src/Codeception/Util/HttpCode.php

```diff
@@ -0,0 +1,162 @@
+<?php
+namespace Codeception\Util;
+
+/**
+ * Class containing constants of HTTP Status Codes
+ * and method to print HTTP code with its description.
+ *
+ * Usage:
+ *
+ * ```php
+ * <?php
+ * use \Codeception\Util\HttpCode;
+ *
+ * // using REST, PhpBrowser, or any Framework module
+ * $I->seeResponseCodeIs(HttpCode::OK);
+ * $I->dontSeeResponseCodeIs(HttpCode::NOT_FOUND);
+ * ```
+ *
+ *
+ */
+class HttpCode
+{
+    const SWITCHING_PROTOCOLS = 101;
+    const PROCESSING = 102;            // RFC2518
+    const OK = 200;
+    const CREATED = 201;
+    const ACCEPTED = 202;
+    const NON_AUTHORITATIVE_INFORMATION = 203;
+    const NO_CONTENT = 204;
+    const RESET_CONTENT = 205;
+    const PARTIAL_CONTENT = 206;
+    const MULTI_STATUS = 207;          // RFC4918
+    const ALREADY_REPORTED = 208;      // RFC5842
+    const IM_USED = 226;               // RFC3229
+    const MULTIPLE_CHOICES = 300;
+    const MOVED_PERMANENTLY = 301;
+    const FOUND = 302;
+    const SEE_OTHER = 303;
+    const NOT_MODIFIED = 304;
+    const USE_PROXY = 305;
+    const RESERVED = 306;
+    const TEMPORARY_REDIRECT = 307;
+    const PERMANENTLY_REDIRECT = 308;  // RFC7238
+    const BAD_REQUEST = 400;
+    const UNAUTHORIZED = 401;
+    const PAYMENT_REQUIRED = 402;
+    const FORBIDDEN = 403;
+    const NOT_FOUND = 404;
+    const METHOD_NOT_ALLOWED = 405;
+    const NOT_ACCEPTABLE = 406;
+    const PROXY_AUTHENTICATION_REQUIRED = 407;
+    const REQUEST_TIMEOUT = 408;
+    const CONFLICT = 409;
+    const GONE = 410;
+    const LENGTH_REQUIRED = 411;
+    const PRECONDITION_FAILED = 412;
+    const REQUEST_ENTITY_TOO_LARGE = 413;
+    const REQUEST_URI_TOO_LONG = 414;
+    const UNSUPPORTED_MEDIA_TYPE = 415;
+    const REQUESTED_RANGE_NOT_SATISFIABLE = 416;
+    const EXPECTATION_FAILED = 417;
+    const I_AM_A_TEAPOT = 418;                                               // RFC2324
```

Comment:
😂 awesome easter egg, but since 418 is in the HTCPCP and not HTTP/1 I think it's kinda inaccurate to have it in a class that is named `HttpCode`.